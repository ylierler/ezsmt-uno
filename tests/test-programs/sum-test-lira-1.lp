&logic(lira).
&type{x;y}=int.
&sum{"2.4"*2;3+x+(5+2)*z}=y.