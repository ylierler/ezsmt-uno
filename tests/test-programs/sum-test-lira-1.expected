{}
z=(/ (- 39) 35) x=0 y=0